src/components/Workspace.jsx

```diff
@@ -250,7 +250,7 @@ class Workspace extends React.Component {
     }
 
     return (
-      <div className="environment">
+      <main className="environment">
```

Comment:
I think the `<div>` with `className="layout"` below may be the most appropriate `<main>` element? LMK your thoughts

Reply:
Hi @outoftime: If we make `<div className="layout">` the `<main>`, the `<header>` would end up inside `<main>`.  

Since Popcode is a single-page site, this is fine. But if the site expands into multiple pages, it may be better to separate `<header>` from `<main>`. Each page would probably share a similar `<header>` but have different `<main>` contents. 

Just something to consider in the future. For now, I can change the `<div className="layout">` to `<main>` as you suggested! LMK what you think.

Reply:
Hey—you’re totally right that `.layout` isn’t a good `<main>` element since it would contain the `<header>`.

Maybe `.layout__columns`, though? Then the `<header>` and `<main>` would be siblings, which seems appropriate?

LMK what you think!

Reply:
You're right! `.layout__columns` works much better as `<main>`. I'll make the changes now.